src/utils.ts

```diff
@@ -37,8 +37,16 @@ export const getHostIpAddress = () => {
 export const getParsedPathname = (
   pathname: string,
   extraMasks?: Array<RegExp>,
-  replacement: string = ":id"
+  replacement: string = ':id'
 ) => {
   const parser = new UrlValueParser({ extraMasks });
   return parser.replacePathValues(pathname, replacement);
 };
+
+export const getSanitizedPath = (pathname: string) => {
+  const sanitizedPath = pathname.replace(
+    /(\/[^?#]+)(?:\?[^#]*)?(?:#.*)?$/,
```

Comment:
@chtushar, Can you add an explanatory comment for this regex? Add an example of string that matches this as well.